Player.js

```diff
@@ -0,0 +1,18 @@
+const PLAYER_VERSION = '0.1';
+
+class Player {
+  static getVersion() {
```

Comment:
I agree with that, this could be: 
`Player.version = '0.1';`
after the class declaration